components/brave_new_tab_ui/containers/newTab/settings.tsx

```diff
@@ -61,9 +59,10 @@ export interface Props {
   showBinance: boolean
   binanceSupported: boolean
   togetherSupported: boolean
+  focusMoreCards: boolean
 }
 
-type ActiveTabType = 'BackgroundImage' | 'BraveStats' | 'Rewards' | 'TopSites' | 'Clock' | 'moreCards'
+type ActiveTabType = 'BackgroundImage' | 'BraveStats' | 'TopSites' | 'Clock' | 'moreCards'
```

Comment:
nit: why the case difference?

Reply:
leftover from when we did dynamic locale filling, updated